nixos/modules/services/x11/xserver.nix

```diff
@@ -749,6 +749,8 @@ in
     boot.kernel.sysctl."fs.inotify.max_user_instances" = mkDefault 524288;
     boot.kernel.sysctl."fs.inotify.max_user_watches" = mkDefault 524288;
 
+    programs.gnupg.agent.pinentryPackage = lib.mkDefault pkgs.pinentry-gnome3;
```

Comment:
One thing to consider is the behaviour when multiple DEs are installed.

Reply:
Hmm, I think distinct priorities for every different pinentry would be a bit silly.  Maybe go back to define the default in the gnupg module?